twitter.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class BasketballTwitterMatch():
+    def __init__(self):
+        uri_base = 'http://www.basketball-reference.com/friv/twitter.cgi'
+        rv = requests.get(uri_base)
+        self.soup = BeautifulSoup(rv.text)
+
+    def gen_player_twitters(self):
+    	twitter_list = self.soup.find('div', {'id': 'div_stats'})
+    	headers = [th.text for th in twitter_list.find_all('th')]
+        rows = [row for row in twitter_list.tbody.find_all('tr')]
```

Comment:
I'm confused: the indentation is obviously wrong in the commit but when I visit the file on my repo, it looks fine. [file in repo](https://github.com/nhatbui/basketball-reference/blob/master/twitter.py)